packages/@o3r/eslint-config/schematics/ng-add/eslint/index.spec.ts

```diff
@@ -15,20 +15,121 @@ import {
 
 const collectionPath = path.join(__dirname, '..', '..', '..', 'collection.json');
 const context = { description: { path: __dirname } };
+const runner = new SchematicTestRunner('schematics', collectionPath);
+
+const monorepoPkgName = '@scope/monorepo';
+const appName = 'test-app';
+const appRoot = `apps/${appName}`;
+const libName = 'test-lib';
+const libRoot = `libs/${libName}`;
+let initialTree: Tree;
+const angularJsonContent = {
+  projects: {
+    [appName]: {
+      root: appRoot,
+      projectType: 'application',
+      architect: {
+        build: {
+          fakeContent: 'should not me modified'
+        }
+      }
+    },
+    [libName]: {
+      root: libRoot,
+      projectType: 'library',
+      architect: {
+        build: {
+          fakeContent: 'should not me modified'
+        }
+      }
+    }
+  }
+};
 
 describe('update eslint config', () => {
-  it('should add a eslint config on workspace', async () => {
-    const initialTree = Tree.empty();
-    initialTree.create('package.json', JSON.stringify({ name: '@scope/package' }));
-    const runner = new SchematicTestRunner('schematics', collectionPath);
+  beforeEach(() => {
+    initialTree = Tree.empty();
+    initialTree.create('package.json', JSON.stringify({ name: monorepoPkgName }));
+    initialTree.create('angular.json', JSON.stringify(angularJsonContent));
+  });
 
-    const tree = await firstValueFrom(callRule(updateEslintConfig(), initialTree, runner.engine.createContext(context as any)));
+  it('should add a eslint config on workspace', async () => {
```

Comment:
```suggestion
  it('should add an eslint config on workspace', async () => {
```
(same for titles of other tests below)